src/Automation/AxeWindowsAutomationException.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+
+namespace Axe.Windows.Automation
+{
+    /// <summary>
+    /// The exception type thrown for all unhandled errors in Axe.Windows.Automation.
+    /// If an exception was thrown from code not owned by Axe.Windows.Automation,
+    /// that exception will be wrapped in <see cref="AxeWindowsAutomationException"/>
+    /// In the <see cref="Exception.InnerException"/> property.
```

Comment:
>In the <see cref="Exception.InnerException"/> [](start = 8, length = 45)

This seems to have some unintended duplication from the previous line

Reply:
I'm afraid I don't see what's duplicated.

Reply:
Ah--the capitalization threw me. Since "In" was capitalized, it sounded like an incomplete thought. Given that InnerException is the standard way of wrapping an Exception, I wonder if it might be less confusing to simply end with 

"that exception will be wrapped in an <see cref="AxeWindowsAutomationException">